giskard-server/src/test/java/tablesaw/TableSawIT.java

```diff
@@ -0,0 +1,40 @@
+package tablesaw;
+
+import ai.giskard.IntegrationTest;
+import org.junit.jupiter.api.Test;
+import tech.tablesaw.api.StringColumn;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+import tech.tablesaw.selection.Selection;
+
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Paths;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+@IntegrationTest
+public class TableSawIT {
+
+    /**
+     * Filter table on stream
+     *
+     * @throws Exception
+     */
+    @Test
+    void filterStream() throws Exception {
+        String location = Paths.get("src/test/bucket/my-dataset.csv").toAbsolutePath().toString();
+        InputStreamReader reader = new InputStreamReader(
+            new FileInputStream(location));
+
+        Table table = Table.read()
+            .usingOptions(CsvReadOptions.builder(reader));
```

Comment:
FYI https://github.com/jtablesaw/tablesaw/issues/290

when you have an input stream as an input TableSaw stores it all in memory, but if you pass a `File` it doesn't

Reply:
you can try running this test with something like 5GB csv